From 6b2d2e0a71d3ebbb608c79f9ff59302cbcc3569e Mon Sep 17 00:00:00 2001
From: Brad Geltz <brad.geltz@intel.com>
Date: Wed, 7 Oct 2020 11:24:14 -0700
Subject: [PATCH 2/2] Use build environment vars
#
#  Copyright (c) 2015, 2016, 2017, 2018, 2019, 2020, Intel Corporation
#
#  Redistribution and use in source and binary forms, with or without
#  modification, are permitted provided that the following conditions
#  are met:
#
#      * Redistributions of source code must retain the above copyright
#        notice, this list of conditions and the following disclaimer.
#
#      * Redistributions in binary form must reproduce the above copyright
#        notice, this list of conditions and the following disclaimer in
#        the documentation and/or other materials provided with the
#        distribution.
#
#      * Neither the name of Intel Corporation nor the names of its
#        contributors may be used to endorse or promote products derived
#        from this software without specific prior written permission.
#
#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
#  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
#  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
#  A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
#  OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
#  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
#  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
#  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
#  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
#  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY LOG OF THE USE
#  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
#

- Updated AVX2 and SKX setup for IMPI_IOMP.

Signed-off-by: Brad Geltz <brad.geltz@intel.com>
---
 setup/Make.IMPI_IOMP_AVX2 | 2 +-
 setup/Make.IMPI_IOMP_SKX  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup/Make.IMPI_IOMP_AVX2 b/setup/Make.IMPI_IOMP_AVX2
index 38b0f73..bff2726 100644
--- a/setup/Make.IMPI_IOMP_AVX2
+++ b/setup/Make.IMPI_IOMP_AVX2
@@ -134,7 +134,7 @@ HPCG_DEFS     = -DMPICH_IGNORE_CXX_SEEK $(HPCG_OPTS) $(HPCG_INCLUDES)
 # - Compilers / linkers - Optimization flags ---------------------------
 # ----------------------------------------------------------------------
 #
-CXX          = mpiicpc
+CXX          = $(MPICXX)
 CXXFLAGS     = -xCORE-AVX2 -qopenmp -std=c++11 $(HPCG_DEFS)
 ifeq (yes, $(DBG))
   CXXFLAGS  += -O0 -g -DHPCG_DEBUG
diff --git a/setup/Make.IMPI_IOMP_SKX b/setup/Make.IMPI_IOMP_SKX
index 9325d71..edc99cd 100644
--- a/setup/Make.IMPI_IOMP_SKX
+++ b/setup/Make.IMPI_IOMP_SKX
@@ -134,7 +134,7 @@ HPCG_DEFS     = -DMPICH_IGNORE_CXX_SEEK $(HPCG_OPTS) $(HPCG_INCLUDES)
 # - Compilers / linkers - Optimization flags ---------------------------
 # ----------------------------------------------------------------------
 #
-CXX          = mpiicpc
+CXX          = $(MPICXX)
 CXXFLAGS     = -xCORE-AVX512 -qopt-zmm-usage=high -qopenmp -std=c++11 $(HPCG_DEFS)
 ifeq (yes, $(DBG))
   CXXFLAGS  += -O0 -g -DHPCG_DEBUG
-- 
2.23.0

